### DISCUSSION TITLE
How to use counter sequence instead of timestamps for sync.

### DISCUSSION THREAD

**Discussion by ubcamit** [Nozbe/WatermelonDB#1042] [Q&A] [answered]
I am planning to use sync WatermelonDB with AzureSQL as the remoteDB. AzureSQL (SQL Server) supports ChangeTracking which generates a versionNumber for each change to the database.  Using the changeTracking features of AzureSQL provides me with multiple benefits: 

1. removes the need to create and update a last_modified field in each table (in the remote DB). 
2. removes the need to store deleted IDs
3. removes the need to handle consistency issues

Could you please guide me on how to replace lastPulledAt with a versionNumber in the front end code (source code) so that the front end can send and receive a versionNumber (int) instead of the lastPulledAt timestamp.

Thank You.

**Comment by radex** [comment, marked as answer]
Read @fahrinh's post: https://fahri.id/posts/how-to-build-watermelondb-sync-backend-in-elixir/

**Reply by ubcamit** [reply]
Thanks a lot. Will go through it and revert if I need some more help.
